include/triton/Conversion/TritonGPUToLLVM/TargetInfoBase.h

```diff
@@ -74,6 +74,15 @@ class TargetInfoBase {
                           StringRef message, StringRef file, StringRef func,
                           int line) const = 0;
 
+  // Whether to enable linear layout. This is a per-backend temporary escape
```

Comment:
Does this disable LLs *everywhere* for AMD?  Or does it just disable this particular feature added in this PR?  The comment suggests we disable it everywhere, but I don't think that's what we want to do.